### PR TITLE
January 12 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

2023 Release Notes have been moved to the archive folder

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 46afcc5 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-384.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by robertengaxway — APPROVED** [review]
I approve

File: content/en/docs/amplify_relnotes/20240112_amplify.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+* **Advanced search filter** - when product is published to a Marketplace, the system index all the markdown documnent related to the product as well as the resource specifictations. This allow to search inside product title, description, documentation, resources specifications or assigned categories. More information [here](/docs/manage_marketplace/consumer_experience/searching#product-search).
+* **Unlimited credential view during the 3 days time frame** - It is now possible to view your credentials from the Marketplace as many time as you want during the timne period those credentials are accessible in the Marketplace (3 days). After those 3 days, the credentials are no more available from the Marketplace, be sure to store them in a safe place.
+
+#### Bug fixes for Marketplace
+
+* **new created application credentials option** - It is now possible, when asking fdor credentials, to reuse an existing application that already have access to the resource.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240112_amplify.md:211]: Typos and grammar

File: content/en/docs/amplify_relnotes/20240112_amplify.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+* **Product resources search fails** - the resource search from the product resource list allows to search either a resource name or a resource groupd name.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240112_amplify.md:212]: **Product resource search** - the search bar on the product resource screen has been fixed to allow searching either by the resource title or the resource group title.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

**Pull request merged**